Cargo.toml

```diff
@@ -25,6 +25,15 @@ embedded-hal = "0.2"
 nb = "0.1"
 stm32f3 = "0.10"
 
+[dependencies.embedded-hal-can]
+version = "0.1.0"
+optional = true
+
+[dependencies.arrayvec]
+version = "0.5.1"
+default-features = false
+optional = true
```

Comment:
[heapless](https://crates.io/crates/heapless) is an alternative and more widely used in the embedded community, FYI. 

But I do not care and haven't used both enough to have an opion, if one or the other would be better for this use case.